_i18n/es.yml

```diff
@@ -72,30 +72,27 @@ index:
       Nuestro objetivo es hacer que el manejo de beneficios, servicios y aplicaciones
       federales sea más fácil y más seguro.
   column_1:
-    heading: Simple *y* seguro para el público
+    heading: Servicio compartido
     content:
-      p_1: Debido a que login.gov es un servicio compartido,
-      li_1: |2-
-
-        Los usuarios necesitan menos contraseñas y aprenden menos interfaces.
-      li_2: Los expertos en seguridad protegen un servicio en lugar de muchos.
-      li_3: Equipos dedicados de expertos en diseño y seguridad lo mejorarán continuamente.
+      p_1: ''
```

Comment:
Is this intentionally blank? Looks like it's showing the English version instead since there is no value here.